.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -30,6 +30,10 @@ about: Report broken or incorrect behaviour
 
 ### System Information
 
-- discord.py version:
-- Python version:
-- Operating system:
+<!-- Do `python -m discord -v` and paste this information.-->
+
+- Python
```

Comment:
Should there be colons after these lines? I also think that the word "version" should still be included as well.